protocol/oracle-manager/contracts/nodes/StalenessCircuitBreakerNode.sol

```diff
@@ -31,6 +31,10 @@ library StalenessCircuitBreakerNode {
             runtimeValues
         );
 
+        if (block.timestamp <= priceNodeOutput.timestamp) {
```

Comment:
It looks like this if statement and the if statement after words could be merged

Reply:
yeah simplified with
```js
block.timestamp - stalenessTolerance <= priceNodeOutput.timestamp
```
As `block.timestamp - stalenessTolerance` cannot get negative.